modules/lazyload/module.php

```diff
@@ -90,13 +90,20 @@ private function apply_dominant_color_background( $control, $value, $selector )
 	public function __construct() {
 		parent::__construct();
 
-		add_action( 'elementor/element/after_add_attributes', function( Element_Base $element ) {
-			$this->update_element_attributes( $element );
-		} );
+		add_action('elementor/frontend/after_enqueue_styles', function() {
+
+			if ( ! \Elementor\Plugin::$instance->documents->get( get_the_ID() )->get_property( 'disable_lazy_load' ) ) {
```

Comment:
Please use a positive property, like the others, e.g. `disable_lazy_load`

Reply:
Done